src/ctia/entity/casebook.clj

```diff
@@ -130,7 +128,8 @@
     :delete-casebook
     :search-casebook})
 
-(def casebook-operation-routes
+(s/defn casebook-operation-routes [{{:keys [read-store write-store]} :StoreService
+                                    :as _services_} :- APIHandlerServices]
```

Comment:
```suggestion
(s/defn casebook-operation-routes [{{:keys [read-store write-store]} :StoreService} :- APIHandlerServices]
```

Reply:
Keeping the `_services_` makes this process a little easier for me.